primitives/consensus/common/src/metrics.rs

```diff
@@ -34,6 +39,42 @@ impl Metrics {
 				)?,
 				registry,
 			)?,
+			block_verification_time: register(
+				HistogramVec::new(
+					HistogramOpts::new(
+						"block_verification_time",
+						"Histogram of time taken to import block",
+					),
```

Comment:
By default the Prometheus client library will use the following buckets:

`0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1.0, 2.5, 5.0, 10.0,` [see crate](https://docs.rs/prometheus/0.7.0/src/prometheus/histogram.rs.html#33-35).

I am not familiar with the timings here. Is this a good fit for this histogram?

Reply:
Yeah, whatever is under 5ms is not very important 

I think minimum block imported is around 10ms